### DISCUSSION TITLE
Sitronix ST1633 I2C Touch controller support

### DISCUSSION THREAD

**Discussion by caipifrosch** [lovyan03/LovyanGFX#445] [General]
Has anyone already integrated ST1633 touch controller in LovyanGFX and is willing to share files or contribute to LoyyanGFX? Help would be appreciated to propperly integrate controller in Loyyan.

**Comment by tobozo** [comment]
there's a [rust driver](https://github.com/embedded-drivers/sitronix-touch) for the sitronix product family, it seems to query the controller to get the capabilities so implementing from this could unlock 11 model ranges with just one codebase

**Reply by caipifrosch** [reply]
Thanks a lot for pointing to this, tobozo! Will give me valuable guidance when writing own driver. Let's see if someone already integrated this in LoveyanGFX.
